.github/workflows/containerd.yml

```diff
@@ -0,0 +1,91 @@
+name: Containerd CI
```

Comment:
suggest name critest

Reply:
Ok, changed it. Lets see what you think.